utils/plugins.js

```diff
@@ -57,6 +104,11 @@ class Plugins {
                 }
             } catch (e) {
                 log.error(e.stack);
+
+                // Auto help
+                if (typeof(cmd) !== 'undefined' && cmd.opts.auto_help){
```

Comment:
`cmd` should be `this[args[0]]`, also the undefined check is un-needed since there's one at line 91

Reply:
This block should be moved to line 100